client/packages/plugins/backendCommon/generated/SyncMessageRow.ts

```diff
@@ -0,0 +1,14 @@
+// This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+import type { SyncMessageRowStatus } from './SyncMessageRowStatus';
+import type { SyncMessageRowType } from './SyncMessageRowType';
+
+export type SyncMessageRow = {
+  id: string;
+  to_store_id: string | null;
+  from_store_id: string | null;
```

Comment:
I guess it would be a breaking change at this point, but would be nice if these got mapped to camel case in TS conversion 👀